easybuild/easyconfigs/p/Python/Python-2.7.9-goolf-1.7.20.eb

```diff
@@ -0,0 +1,119 @@
+name = 'Python'
+version = '2.7.9'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.9.1'
+scipyversion = '0.14.1'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+#    ('SQLite', '3.8.8.1'),
+#   ('OpenSSL', '1.0.1k'),  # OS dependency should be preferred if the os version is more recent then this version, it's
+#   nice to have an up to date openssl for security reasons
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+# order is important!
+# package versions updated Jan 19th 2015
+exts_list = [
+    ('requests', '2.6.2', {
```

Comment:
we don't have this one in the other easyconfigs (yet)

Is it a strict requirement for other stuff?

I'd hate to have to sync up all the easyconfigs for this Python version, especially since 2.7.10 is there too.